app/org/sagebionetworks/bridge/models/RequestInfo.java

```diff
@@ -0,0 +1,205 @@
+package org.sagebionetworks.bridge.models;
+
+import java.util.LinkedHashSet;
+import java.util.Objects;
+import java.util.Set;
+
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+
+import org.sagebionetworks.bridge.models.studies.StudyIdentifier;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Information about the criteria and access times of requests from a specific user. Useful for 
+ * support and troubleshooting, and potentially useful to show the filtering that is occurring on 
+ * the server to researchers in the researcher UI.
+ */
+public final class RequestInfo {
+
+    private final String userId;
+    private final ClientInfo clientInfo;
+    private final String userAgent;
+    private final LinkedHashSet<String> languages;
+    private final Set<String> userDataGroups;
+    private final DateTime activitiesAccessedOn;
+    private final DateTime signedInOn;
+    private final DateTimeZone timeZone;
+    private final StudyIdentifier studyIdentifier;
+
+    @JsonCreator
+    private RequestInfo(@JsonProperty("userId") String userId, @JsonProperty("clientInfo") ClientInfo clientInfo,
+            @JsonProperty("userAgent") String userAgent, @JsonProperty("languages") LinkedHashSet<String> languages,
+            @JsonProperty("userDataGroups") Set<String> userDataGroups,
+            @JsonProperty("activitiesAccessedOn") DateTime activitiesAccessedOn,
+            @JsonProperty("signedInOn") DateTime signedInOn, @JsonProperty("timeZone") DateTimeZone timeZone,
+            @JsonProperty("studyIdentifier") StudyIdentifier studyIdentifier) {
+        this.userId = userId;
+        this.clientInfo = clientInfo;
+        this.userAgent = userAgent;
+        this.languages = languages;
+        this.userDataGroups = userDataGroups;
+        this.activitiesAccessedOn = activitiesAccessedOn;
+        this.signedInOn = signedInOn;
+        this.timeZone = timeZone;
+        this.studyIdentifier = studyIdentifier;
+    }
+
+    public String getUserId() {
+        return userId;
+    }
+
+    public ClientInfo getClientInfo() {
+        return clientInfo;
+    }
+    
+    public String getUserAgent() {
+        return userAgent;
+    }
+
+    public LinkedHashSet<String> getLanguages() {
+        return languages;
+    }
+
+    public Set<String> getUserDataGroups() {
+        return userDataGroups;
+    }
+
+    public DateTime getActivitiesAccessedOn() {
+        return (timeZone != null && activitiesAccessedOn != null) ?
+                activitiesAccessedOn.withZone(timeZone) : activitiesAccessedOn;
+    }
```

Comment:
We should probably add tests to make sure getActivitiesAccessedOn behaves reasonably when the DateTime differs from the TimeZone (not likely with getActivitiesAccessedOn, but very likely for getSignedInOn). We should also test to make sure the behavior is reasonable for *no* timezone.